contracts/contracts/harvest/OETHHarvesterSimple.sol

```diff
@@ -55,8 +69,13 @@ contract OETHHarvesterSimple is Strategizable {
             // Cache balance
             uint256 balance = IERC20(rewardTokens[i]).balanceOf(address(this));
             if (balance > 0) {
+                // Determine receiver
+                address receiver = rewardTokens[i] == WETH
+                    ? dripper
+                    : strategistAddr;
+
                 // Transfer to strategist
-                IERC20(rewardTokens[i]).safeTransfer(strategistAddr, balance);
+                IERC20(rewardTokens[i]).safeTransfer(receiver, balance);
                 emit Harvested(_strategy, rewardTokens[i], balance);
```

Comment:
nit: Let's also emit the receiver?

Reply:
Added in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2363/commits/4343e2e26360fb7ddd0ad5759d736e869029c696.